### PR TITLE
chore: address connector feedback

### PR DESCRIPTION
**Pull request by kyungeunni**
ticket: https://ubio-automation.atlassian.net/browse/ROBO-511

- [x] Move the Auth selection component to the top
- [x] Change Object.compose to Value.getJson for better readability
- [x] include API documentation link(root) to each generated action’s help
- [x] When creating the connector action, remember the last selected auth and apply (if possible)
- [x] parse response body

### CODE REVIEW

File: packages/engine/src/main/connector.ts [hunk 1]
```diff
@@ -65,25 +66,139 @@ export function buildConnectors(namespace: string, spec: ConnectorSpec) {
         if (!valid) {
             continue;
         }
-        const name = `${namespace}.${endpoint.name}.${endpoint.method.toLocaleLowerCase()}`;
-        class ChildConnectorAction extends ConnectorAction {
-            static $type = name;
-            static $help = endpoint.description;
+        const type = `${namespace}.${endpoint.name}.${endpoint.method.toLocaleLowerCase()}`;
+        class ConnectorAction extends Action {
```
Comment (kyungeunni) [inline packages/engine/src/main/connector.ts:70]: I just inlined ConnectorAction declaration here as having them separately then inheriting it again increased complexity as we needed to figure the params order and it also introduced meaningless abstract methods which didn't do much but just increased complexity in my opinion.

File: packages/autopilot/src/app/viewports/script-editor/params/param-credentials.vue [hunk 2]
```diff
@@ -55,6 +55,16 @@ export default {
 
     mixins: [ParamMixin],
 
+    mounted() {
+        if (!this.value) {
+            this.setLastSelected();
```
Comment (inca) [inline packages/autopilot/src/app/viewports/script-editor/params/param-credentials.vue:60]: Oh nice. I was afraid it'd be too tricky to implement — but your approach here looks good 👍

File: packages/engine/src/main/connector.ts (hunk 1 above)
Comment (inca) [reply packages/engine/src/main/connector.ts:72]: Yeah, I _kinda_ agree. The only _real_ problem is that the whole class is created in a closure which generally means that it has access to its outer scope (i.e. including call arguments of the function) — so I'm still keen on flattening it as much as we can (for this exact reason we strongly advise against nested function declarations).

Comment (inca) [reply packages/engine/src/main/connector.ts:72]: I think there are few options though. We can 🤙 if you'd like

**Review by inca — COMMENTED** [review]
No major objections — but let's discuss what we can do with the inlined class.

**Review by inca — APPROVED** [review]
Looks food 🚀

**Pull request merged**